csb/src/main/scala/edu/msstate/dasi/csb/DataParser.scala

```diff
@@ -20,14 +27,14 @@ object DataParser {
   }
 
 
-  def logToGraph(augLogPath: String, partitions: Int): Graph[VertexData, EdgeData] = {
-    val augLogFile = sc.textFile(augLogPath, partitions)
+  def logToGraph(logPath: String, partitions: Int): Graph[VertexData, EdgeData] = {
+    val logFile = sc.textFile(logPath, partitions)
 
-    // Drop the 8-lines header and filter lines that contains only IPv4 addresses
-    val augLog = augLogFile.mapPartitionsWithIndex { (idx, lines) => if (idx == 0) lines.drop(8) else lines }
+    // Drop the 8-lines header and 1-line footer and filter lines that contains only IPv4 addresses
+    val theLog = logFile.filter(isNotComment)
       .filter(isInet4).filter(isAllowedProto)
```

Comment:
Good catch! Since it is a simple check, I would replace the isNotComment function with an anonymous function and also all filters can be placed together on one line:
````scala
val theLog = logFile.filter(line => line(0) != '#').filter(isInet4).filter(isAllowedProto)